Sources/GRPC/TLSVerificationHandler.swift

```diff
@@ -38,26 +37,21 @@ public class TLSVerificationHandler: ChannelInboundHandler, RemovableChannelHand
     return verificationPromise.futureResult
   }
 
-  public init(errorDelegate: ClientErrorDelegate?) {
```

Comment:
Why was the error delegate removed? Is that done through `context.fireErrorCaught(error)` now?

Reply:
Exactly. We added an error handler with the error delegate after this a little while ago and never removed this.